### ISSUE TITLE
Maintenance status / help needed?

### ISSUE THREAD

**Issue by maelle** [ropensci/pangaear#80]
@gavinsimpson @naupaka  

Do you still intend to become this package's maintainers?

If so do you need any help? For instance an aspect where you'd appreciate some tips, contributions, a PR review? Do you need an invitation to our friendly Slack workspace?

**Comment by gavinsimpson** [comment]
@maelle yes, that is still my intention. I'm paying the price of being out of the office too much this summer and now need to find time in my schedule to dig back into pangaear as it's been a while since I looked at the package code. Issue #79 looks like it will require some not insignificant changes to the way we query panagaea itself so I need to get back up to speed with good R ways for querying R APIs.

Access to Slack would be good - I can't recall but I think I had some level of ropensci slack access but I have been muting it for a while as I'm signed in to too many slack groups and need to do a prune.

@naupaka and I should probably arrange a quick call in the near future to see how we want to proceed.

**Comment by maelle** [comment]
Thanks for the update!

What email address should our community manager @yabellini use for the slack invite? (hers is `yabellini@ropensci.org` if you prefer not to write an email address here) (same question to you @naupaka)

Happy to help with questions related to R and APIs :smile_cat: (Scott and I updated https://books.ropensci.org/http-testing a while ago)

**Comment by naupaka** [comment]
@maelle yes I am in the same boat as @gavinsimpson. I'm keen to continue to be involved, but after a summer of field work and conference travel I'm also working to get back on top of my GH notification backlog. I think I'm on the ROS Slack workspace already?

**Comment by maelle** [comment]
Thanks @naupaka, yes I can see you in the slack member list.
